interface/wx/app.h

```diff
@@ -1068,6 +1068,21 @@ class wxApp : public wxAppConsole
     */
     virtual bool OSXIsGUIApplication();
 
+    /**
+        Enable the automatic tabbing features of OSX.
```

Comment:
```suggestion
        Enable the automatic tabbing features of macOS.
```
As this is the new official OS name.

Reply:
Thanks for pointing that out. I fixed it in the other places it was referenced as well.